install-php-extensions

```diff
@@ -854,6 +861,12 @@ compareVersions() {
 installBundledModule() {
 	printf '### INSTALLING BUNDLED MODULE %s ###\n' "$1"
 	case "$1" in
+		enchant)
+			if test "$DISTRO" = "alpine" && test $DISTRO_VERSION_INT -ge 312 && test $PHP_MAJMIN_VERSION -lt 800; then
+				echo "Cannot install enchant on $DISTRO_VERSION with php $PHP_MAJMIN_VERSION - https://bugs.php.net/bug.php?id=76962"
```

Comment:
This is not very convincing, maybe there is a better idea @mlocati ? Basically on alpine 3.12 there is only enchant2, but php<8 doesnt support enchant2, so on this version of alpine its impossible to enable enchant